src/playerClass.ts

```diff
@@ -12,11 +14,34 @@ export class Player {
     }
     this._name = name;
     this._pokemons = pokemons;
+    this._activePokemon = 0;
+    this._alivePokemons = pokemons;
   }
+
   get name(): string {
     return this._name;
   }
   get pokemons(): Pokemon[] {
     return this._pokemons;
   }
+
+  set setActivePokemon(numOfActivePokemon: number) {
+    this._activePokemon = numOfActivePokemon;
+  }
+  get getActivePokemon(): number {
+    return this._activePokemon;
```

Comment:
Maybe at once, we can return the Pokemon object, not only index, because this is what we will need anyway? And probably we do not need to repeat get and set in the name :)
`return this._pokemons[this._activePokemon];`

Reply:
Yes, it's true. It will be better